lib/manifest.js

```diff
@@ -0,0 +1,56 @@
+"use strict";
+
+let retryTimings = [10, 50, 100, 250, 500, 1000];
+
+module.exports = class Manifest {
+	constructor() {
+		this._index = {};
+
+		this._lookup = this._lookup.bind(this);
+	}
+
+	lookup(targetPath) {
+		return retry(this._lookup, retryTimings)(targetPath);
+	}
+
+	_lookup(targetPath) {
+		return new Promise((resolve, reject) => {
+			let outputPath = this.manifest[targetPath];
```

Comment:
I'm confused. What is `this.manifest`? Shouldn't this be `let outputPath = this.get(targetPath)`?